timesketch/frontend-ng/src/components/Studio/SigmaRuleModification.vue

```diff
@@ -0,0 +1,233 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-container class="px-8">
+    <h2>
+      {{ editingRule.title }}
+      <v-chip rounded x-small class="mr-2" :color="statusColors()">
+        <v-icon v-if="isParsingSuccesful" x-small> mdi-check </v-icon>
+        <v-icon v-else x-small> mdi-alert </v-icon>
+        {{ status_chip_text }}</v-chip
+      >
+    </h2>
+
+    <div>
+      <v-autocomplete dense filled rounded :items="SigmaTemplates" @change="selectTemplate" item-text="title">
```

Comment:
Stay consistent with other autocompletes - use `outlined` and remove "filled and rounded".

Aim for something like this:

<img width="772" alt="Screen Shot 2023-01-13 at 13 20 34" src="https://user-images.githubusercontent.com/316362/212319075-747b4e11-1e6c-4061-b313-bab7ad12e0ca.png">

Reply:
Ok did that, and now have a default label, but the strong header is font size to small, WDYT?

Reply:
and I cut the width of the template choose smaller with `<v-col cols="4">` WDYT?